viewer/core/src/public/migration/Cognite3DModel.ts

```diff
@@ -39,6 +40,35 @@ export class Cognite3DModel extends THREE.Object3D implements CogniteModelBase {
     return this._styledNodeCollections;
   }
 
+  /**
+   * Returns the unit the coordinates for the model is stored in. Possible return values are:
+   * - Meters
+   * - Centimeters
+   * - Millimeters
+   * - Micrometers
+   * - Kilometers
+   * - Feet
+   * - Inches
+   * - Yards
+   * - Miles
+   * - Mils
+   * - Microinches
+   * Note that coordinates in Reveal always are converted to meters using {@see modelUnitToMetersFactor}.
+   * @version New since 2.1
+   */
+  get modelUnit(): string {
+    return this.cadNode.cadModelMetadata.scene.unit;
```

Comment:
This could potentially also return empty string, which should also be indicated in the jsdoc.

Reply:
Also, this should actually be returned to the user as an enum, right? Or are we afraid of new measurement units to be added?

Reply:
Added a union type for units.